### PR TITLE
Fix local run and deploy templates for HEAD bazel.

### PR DESCRIPTION
**Pull request by pmbethe09**
Get a working 'bazel run test:test-war'.  Use workspace_name where needed in deploy template.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: appengine/appengine.bzl [hunk 1]
```diff
@@ -155,6 +155,8 @@ def _war_impl(ctxt):
       appengine_sdk = f.path
```
Comment (kchodorow) [inline appengine/appengine.bzl]: I think this can be changed to short_path to avoid the path munging on line 159.

File: appengine/appengine.bzl [hunk 2]
```diff
@@ -205,10 +207,10 @@ appengine_war = rule(
             single_file = True,
         ),
         "_appengine_sdk": attr.label(
-            default = Label("//external:appengine/java/sdk"),
+            default = Label("@com_google_appengine_java//:sdk"),
```
Comment (kchodorow) [inline appengine/appengine.bzl]: Might as well remove the bindings while you're here.

File: test/BUILD [hunk 3]
```diff
@@ -3,10 +3,12 @@ load("//appengine:appengine.bzl", "appengine_war")
 appengine_war(
     name = "test-war",
     data = [":test-data"],
-    data_path = "/test",
-    jars = ["libtest.jar"],
+    data_path = "/test/data",
+    jars = [":app_deploy.jar"],
 )
 
+java_binary(name="app", main_class="dummy")
```
Comment (kchodorow) [inline test/BUILD]: Nit: formatting.

**Comment by pmbethe09** [comment]
let me try again -- this seems to have taken the wrong email address in git commit.

**Comment by pmbethe09** [comment]
I signed it!

(by changing to the right email)

**Comment by kchodorow** [comment]
LGTM, thanks!

**Comment by kchodorow** [comment]
Squashed and merged as 8f357212c9dc1aa8fe3a03b082bc2f61c5a18502, fixing #15.
